runtime/common/RuntimeMLIR.cpp

```diff
@@ -71,17 +70,22 @@ TranslateFromMLIRRegistration::TranslateFromMLIRRegistration(
 
 namespace cudaq {
 
+static std::once_flag mlir_init_flag;
+static bool mlirLLVMInitialized = false;
```

Comment:
Do we still need the `mlirLLVMInitialized` flag ?

Reply:
Please get rid of it.